### ISSUE TITLE
Two devices can't connect simultaneously to a client

### ISSUE THREAD

**Issue by stellawills** [Nyr/openvpn-install#1133]
I tried your configuration and many other numerous configuration. I'll need a clarification if it's openvpn that doesn't allow simultaneous connection to a client a something else.

I connected on one of my device with client1.ovpn it's works perfectly allowing Internet access and all but by the rime I connect the same with the same client1.ovpn on another device the first device begins not to access the Internet anymore till I disconnect and Reconnect but the new device connected access the Internet.

TL;DR: Connect simultaneously on two device makes one of the unable to access the Internet until its Reconnect and when Reconnect the later (one already connected) doesn't access the Internet anymore.

I'm trying to build openvpn client for about 100 users simultaneously connected.

**Comment by davinbirdi** [comment]
For each of the 100 users they should have their own .ovpn file/configuration (i.e. client1.ovpn, client2.ovpn, etc... or a more descriptive name).

I am not familiar with the security reasons behind it, but I understand it as a way to recognize if someone snooped your certifications/.ovpn file and attempted to login as you rather than have you both connected representing you.

**Comment by Nyr** [comment]
Indeed it is highly preferrable to use one certificate per client. If you have a good reason to deploy a single certificate to multiple endpoints (which is most often not the case), take a look at the `duplicate-cn` configuration directive.
